e2e/tests/regression/test_replica.robot

```diff
@@ -69,4 +69,45 @@ Test Offline Replica Rebuilding
     And Wait until volume 0 replica rebuilding started on node 0
     And Wait for volume 0 detached
     And Volume 0 should have 3 replicas when detached
-    And Set setting offline-replica-rebuilding to false
\ No newline at end of file
+    And Set setting offline-replica-rebuilding to false
+
+Test Preempt Offline Replica Rebuilding By A Workload
+    [Tags]    coretest    offline-rebuilding
+    [Documentation]    Test preempt offline replica rebuilding by a workload.
+    ...
+    ...                Issue: https://github.com/longhorn/longhorn/issues/8443
+    ...    1. Create a workload with Longhorn volume having 3 replicas in a 3 worker nodes cluster
+    ...    2. Write some data to the volume.
+    ...    3. Scale down the workload to detach the volume.
+    ...    4. Delete a replica of the volume.
+    ...    5. Enable the offline rebuilding by the API `volume.offlineReplicaRebuilding`.
+    ...    6. When the rebuilding is in progress, scale up the workload.
+    ...    7. Check if the offline rebuilding process is canceled.
+    ...    8. Check if the workload is working well.
```

Comment:
The Given, When, Then should be self-explanatory. Consider removing the steps from the documentation section.

Reply:
Removed. Thanks.